eclipse/src/saros/session/resources/validation/ResourceChangeValidator.java

```diff
@@ -60,22 +64,34 @@ public IStatus validateChange(final IResourceDelta delta, final IProgressMonitor
 
     if (currentSession == null) return Status.OK_STATUS;
 
-    final ResourceDeltaVisitor visitor = new ResourceDeltaVisitor(currentSession);
+    Set<IReferencePoint> sharedReferencePoints = session.getReferencePoints();
 
-    try {
-      delta.accept(visitor);
-    } catch (CoreException e) {
-      log.warn(
-          "error occured during delta visiting, some resources might have not been checked", //$NON-NLS-1$
-          e);
-    }
+    for (IReferencePoint referencePoint : sharedReferencePoints) {
+      IContainer referencePointDelegate = ResourceConverter.getDelegate(referencePoint);
+
+      IResourceDelta referencePointDelta = delta.findMember(referencePointDelegate.getFullPath());
+
+      final ResourceDeltaVisitor visitor = new ResourceDeltaVisitor(referencePoint, currentSession);
```

Comment:
Should be place above line 80, not here.

Reply:
Will do.

Reply:
Done.